client/src/app/components/FiatSelector.scss

```diff
@@ -26,6 +26,10 @@
     color: var(--body-color);
     appearance: none;
 
+    @include phone-down {
+      padding: 0 24px 0 8px;
```

Comment:
I think a small decrease in `fiat-selector_button` height, like 35px, would harmonize mobile navbars

Reply:
thanks you, it looks great! 👍🏼